supervisor/docker/interface.py

```diff
@@ -162,6 +162,11 @@ def _install(
                     "Available space in /data is: %s GiB",
                     free_space,
                 )
+            elif err.status_code == 429:
+                _LOGGER.info(
+                    "You are now ratelimited by Docker Hub, for more details see "
+                    "https://www.home-assistant.io/more-info/dockerhub-rate-limit"
+                )
```

Comment:
Yes!
But I want do do that in a separate PR, refactor both these to our new format, and add a job decorator to this method as well

Reply:
With job decorators, we need a bit carefully, since they don't support all needed features. But the Resolution/issue part is fixed, please create an issue. We just refactor later the storage with job decoration on all update/install methods. You need also not create an issue for the storage part as we remove that. Only for this code part.